### PR TITLE
CSS modules

### PR DESCRIPTION
**Pull request by rickboot**

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.spec.js
```
Comment (ryanminjaeshin) [inline .eslintignore]: Should we ignore ESlint for our test?

File: .eslintrc.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "env": {
         "browser": true,
-        "es2020": true,
+        "es2021": true,
```
Comment (ryanminjaeshin) [inline .eslintrc.json]: To not have ESlint ignore on spec, add this line of code in line 5 here.

        "jest": true

File: client/src/components/Answer/Answer.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Answer () {
```
Comment (ryanminjaeshin) [inline client/src/components/Answer/Answer.jsx]: I love all of your straight forward components' names. Great job with naming conventions.

File: client/src/components/Questions/QuestionsList/Question/Question.jsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```
Comment (ryanminjaeshin) [inline client/src/components/Questions/QuestionsList/Question/Question.jsx]: It is great that you layout you components with class. It is easier for us to work on.

File: client/src/components/Questions/QuestionsTab/QuestionsTab.css [hunk 5]
```diff
@@ -0,0 +1,12 @@
+.tab {
```
Comment (ryanminjaeshin) [inline client/src/components/Questions/QuestionsTab/QuestionsTab.css:1]: Your css styling looked perfect in our meeting. Gotta teach me how to do it lol.

File: client/src/components/Questions/QuestionsTab/QuestionsTab.spec.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+import ReactDOM from 'react-dom';
+import QuestionsTab from './QuestionsTab.jsx';
+import { shallow, mount } from 'enzyme';
+// import '../../../setupTests.js';
+
+describe('QuestionsTab unit tests', () => {
+  it ('Renders without crashing', () => {
+    shallow(<QuestionsTab />);
+  });
+  it("Renders 'Reviews' and 'Questions' in tab", () => {
```
Comment (ryanminjaeshin) [inline client/src/components/Questions/QuestionsTab/QuestionsTab.spec.js:11]: Nice basic enzyme testing. Only one thing I like to recommend here is for you to use a single quote. 💯

**Review by ryanminjaeshin — APPROVED** [review]
Great job. I do really like the structure of your components. I can map it out in my head just by looking at it. 👍

File: client/src/components/Questions/QuestionsTab/QuestionsTab.spec.js (hunk 6 above)
Comment (rickboot) [reply client/src/components/Questions/QuestionsTab/QuestionsTab.spec.js:11]: I am nesting quotes so I use ' and ". :)

File: .eslintrc.json (hunk 2 above)
Comment (rickboot) [reply .eslintrc.json]: thanks for the tip. less red is better.

**Pull request closed without merging**